docs/install_guide.adoc

```diff
@@ -0,0 +1,206 @@
+= User Installation Guide
+
+== Getting Started
+
+Before installing the Watcher operator you first need a functional
+OpenShift installation with the required Openstack operators,
+including the Telemetry operator. The following links point
+to documents detailing how to create this required starting environment:
+
+* https://github.com/openstack-k8s-operators/openstack-operator[Openstack Operator]
+* https://github.com/openstack-k8s-operators/telemetry-operator[Telemetry Operator]
+* https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Kubernetes operators]
+* https://prometheus.io/[Prometheus metrics]
+
+A CRC (https://crc.dev/docs/introducing/[Code Ready Containers]) installation is
+adequade for a developer environment. To verify that the environment set up is ready,
+log in to the Kubernetes/Openshift environment:
+
+[,console]
+----
+$ oc login -u <username> -p <password> https://api.crc.testing:6443 --insecure-skip-tls-verify=true
+----
+
+Then access the Openstack client and verify the service endpoints are available:
```

Comment:
Suggest removing `Then` and making this sentence step 2, by prefixing the sentence with a period + whitespace char.

. Access the Openstack client and verify the service endpoints are available:[](url)